cmd/pool/order.go

```diff
@@ -396,6 +401,14 @@ var ordersSubmitAskCommand = cli.Command{
 				"unannounced ones. The default value is \"no " +
 				"preference\"",
 		},
+		cli.Uint64Flag{
+			Name: "confirmation_constraints",
+			Usage: "specifies if the liquidity must be sold in " +
+				"zeroconf or confirmed channels. Set to 1 " +
+				"for only confirmed channels and 2 for only " +
+				"zeroconf ones. The default value is \"no " +
```

Comment:
Could also set the default value here so it would show up correctly. And we need to remove the "The default value is" sentence.

Reply:
modified